integration/command.go

```diff
@@ -69,8 +69,9 @@ var deployInspektorGadget *command = &command{
 }
 
 var waitUntilInspektorGadgetPodsDeployed *command = &command{
-	name: "Wait until the gadget pods are started",
-	cmd:  "for POD in $(sleep 5 ; kubectl get pod -n gadget -l k8s-app=gadget -o name) ; do kubectl wait --timeout=30s -n gadget --for=condition=ready $POD ; done ; kubectl get pod -n gadget",
+	name:           "Wait until the gadget pods are started",
+	cmd:            "ret=\"\" ; for POD in $(sleep 5 ; kubectl get pod -n gadget -l k8s-app=gadget -o name) ; do kubectl wait --timeout=30s -n gadget --for=condition=ready $POD; ret=\"$ret$(echo $?)\" ; done ; kubectl get pod -n gadget ; echo \"ret=$ret\" ; ",
```

Comment:
You can use write:

```
`ret=""...`,
```

To avoid having to escape all inner quotes.

Reply:
I used a different approach.